frontend/server/src/Controllers/Problem.php

```diff
@@ -30,7 +30,8 @@
  * @psalm-type SelectedTag=array{public: bool, tagname: string}
  * @psalm-type ProblemAdmin=array{role: string, username: string}
  * @psalm-type ProblemGroupAdmin=array{alias: string, name: string, role: string}
- * @psalm-type ProblemVersion=array{author: array{email?: string, name?: string, time: \OmegaUp\Timestamp|null}, commit: string, committer: array{email?: string, name?: string, time: \OmegaUp\Timestamp|null}, message?: string, parents?: list<string>, tree: array<string, string>|null, version: null|string}
+ * @psalm-type Signature=array{email?: string, name?: string, time: \OmegaUp\Timestamp|null}
```

Comment:
```suggestion
 * @psalm-type Signature=array{email?: string, name?: string, time: \OmegaUp\Timestamp}
```

así `Signature` es compatible con el tipo que está reemplazando.

Reply:
Dejé otro comentario del porqué no pude quitar esto. ¿Se te ocurre una buena forma de hacer que esto sea no-nulo?